### PR TITLE
New sidebar on projects page with shortcuts to your projects and mentees projects

### PR DESCRIPTION
**Pull request by agundy**
When not logged in:
image
When logged in as a normal user:
image
When logged in as a mentor:
image

### CODE REVIEW

**Comment by kmcnellis** [comment]
I think it would be better to just not include the sidebar when not logged in. It just looks off

**Comment by agundy** [comment]
Updated to show the old way with col-12 if the user is not logged in

File: server/api/project/index.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ router.get('/defaults', controller.defaults)
 router.get('/past', controller.indexOld);
 router.get('/:id/authors', controller.authors);
 router.get('/:username/:project', controller.show);
+router.get('/mine', auth.isAuthenticated(), controller.my_projects);
```
Comment (SaswatB) [inline server/api/project/index.js]: Please use camel to follow convetion

File: server/api/project/project.controller.js [hunk 2]
```diff
@@ -88,6 +91,42 @@ exports.show = function(req, res) {
   }
 };
 
+exports.my_projects = function(req, res) {
```
Comment (SaswatB) [inline server/api/project/project.controller.js]: Camel case here to

File: server/api/user/user.model.js [hunk 3]
```diff
@@ -559,7 +559,7 @@ UserSchema.methods = {
             res.currentSmallAttendance = [];
         }
         return res;
-    }
+    },
```
Comment (SaswatB) [inline server/api/user/user.model.js]: Extra comma

**Comment by agundy** [comment]
@SaswatB updated!

**Pull request merged**